FREEBSD.md

```diff
@@ -0,0 +1,32 @@
+# Building on FreeBSD
+Building on FreeBSD is made possible by cctools-port.
+
+This assumes you have sudo installed and configured.
+
+1. install packages
+```
+sudo pkg install findutils llvm12 git gmake patch gnugrep po4a docbook-xsl ncurses autoconf automake gettext libtool pkgconf dpkg fakeroot zstd python39 coreutils bash cmake gnupg openssl gsed gtar perl5 wget
```

Comment:
Is there a reason you want llvm12 instead of using the clang in world?

Reply:
because clang doesn't exist

Reply:
wdym? clang is part of world, https://cgit.freebsd.org/src/tree/usr.bin/clang

Reply:
```
[nick@nick-pc ~]$ pkg search clang
flang-clang-7.0.g20191020      Fortran compiler targeting LLVM (modified clang part)
opencl-clang-11.0.0            Clang wrapper to compile OpenCL C kernels to SPIR-V modules
[nick@nick-pc ~]$ 
```

Reply:
world isn't tracked by pkg, run `clang --version`

Reply:
```
[nick@nick-pc ~]$ clang --version
FreeBSD clang version 11.0.1 (git@github.com:llvm/llvm-project.git llvmorg-11.0.1-0-g43ff75f2c3fe)
Target: x86_64-unknown-freebsd13.0
Thread model: posix
InstalledDir: /usr/bin
```
so i guess i should remove llvm12?